src/qt/locale/bitcoin_nl.ts

```diff
@@ -1336,19 +1325,19 @@
     </message>
     <message>
         <source>Disables some advanced features but all blocks will still be fully validated. Reverting this setting requires re-downloading the entire blockchain. Actual disk usage may be somewhat higher.</source>
-        <translation type="unfinished"></translation>
+        <translation>Schakelt enkele geavanceerde functies uit maar alle blokken worden nog steeds volledig gevalideerd. Deze instelling terugdraaien vereist dat de gehele blok keten opnieuw gedownload wordt. In realiteit kan de gebruikte diskruimte hoger uitvallen.</translation>
```

Comment:
Schakelt enkele geavanceerde functies uit maar alle blokken worden nog steeds volledig gevalideerd. Deze instelling terugdraaien vereist dat de gehele `blockchain` opnieuw gedownload wordt. `Het werkelijke schijfgebruik kan iets hoger zijn.`

blockchain should not be translated.

Reply:
So I checked, and we currently translate blockchain to "blok keten" everywhere. I'm okay with changing it, but then we change it everywhere.

Reply:
i think its oke to keep "blok keten".